### PR TITLE
Add Cereal Library

### PR DESCRIPTION
**Pull request by DongTrieu1998**

### CODE REVIEW

File: fudy/CMakeLists.txt [hunk 1]
```diff
@@ -8,10 +8,15 @@ set(CMAKE_AUTORCC ON)
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_TOOLCHAIN_FILE "${CMAKE_BINARY_DIR}/conan_toolchain.cmake")
+
+option(BUILD_SHARED_LIBS "Set the default type for add_library to STATIC" OFF)
```
Comment (ghost) [inline fudy/CMakeLists.txt]: Change the description

File: fudy/cmake/conanpackage.cmake [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    elseif(MSVC_TOOLSET_VERSION EQUAL 141)
+        set(COMPILER_VERSION 191)
+    else()
+        message(FATAL_ERROR "Unsupported MSVC toolset version " ${MSVC_TOOLSET_VERSION})
+    endif()
+
+    list(APPEND CONAN_ARGS -s compiler.version=${COMPILER_VERSION})
+    list(APPEND CONAN_ARGS -s:b compiler.version=${COMPILER_VERSION})
+
+
```
Comment (ghost) [inline fudy/cmake/conanpackage.cmake]: Remove line

**Review by ghost — APPROVED** [review]

File: fudy/CMakeLists.txt (hunk 1 above)
Comment (DongTrieu1998) [reply fudy/CMakeLists.txt]: Could you please clarify for me what you mean?

Comment (DongTrieu1998) [reply fudy/CMakeLists.txt]: I updated this description.

File: fudy/cmake/conanpackage.cmake (hunk 2 above)
Comment (DongTrieu1998) [reply fudy/cmake/conanpackage.cmake]: This line was removed.

**Review by sangnguyencs20 — APPROVED** [review]

**Pull request merged**